public/request/user/update-avatar.php

```diff
@@ -17,7 +17,16 @@
     exit;
 }
 
-$response['Response'] = UploadUserPic($user, $filename, $rawImage);
-
+$uploadResponse = UploadUserPic($user, $filename, $rawImage);
+$response['Success'] = $uploadResponse['Success'];
+unset($uploadResponse['Success']);
 settype($response['Success'], 'boolean');
+
+if ($uploadResponse['Error']) {
+    $response['Error'] = $uploadResponse['Error'];
+    unset($uploadResponse['Error']);
+}
+
+$response['Response'] = $uploadResponse;
+
```

Comment:
As this endpoint is only used internally i'd prefer it to be simplified to `$response = UploadUserPic($user, $filename, $rawImage);` as suggested above.

Reply:
done